src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -1204,48 +885,11 @@ def _update_mon_count(current, new):
             return new
         return self._patch(ccl.CephCluster, 'cephclusters', self.rook_env.cluster_name, _update_mon_count)
 
-    def add_osds(self, drive_group, matching_hosts):
-        # type: (DriveGroupSpec, List[str]) -> str
-        assert drive_group.objectstore in ("bluestore", "filestore")
-        assert drive_group.service_id
-        storage_class = self.get_storage_class()
-        inventory = self.get_discovered_devices()
-        creator: Optional[DefaultCreator] = None
-        if (
-            storage_class.metadata.labels
-            and 'local.storage.openshift.io/owner-name' in storage_class.metadata.labels
-        ):
-            creator = LSOCreator(inventory, self.coreV1_api, self.storage_class_name)
-        else:
-            creator = DefaultCreator(inventory, self.coreV1_api, self.storage_class_name)
-        return self._patch(
-            ccl.CephCluster,
-            'cephclusters',
-            self.rook_env.cluster_name,
-            creator.add_osds(self.rook_pods, drive_group, matching_hosts)
-        )
-
-    def remove_osds(self, osd_ids: List[str], replace: bool, force: bool, mon_command: Callable) -> str:
```

Comment:
Same than with OSD creation. I would suggest to raise an exception. (This method is part of the orchestrators standard interface)